### PR TITLE
Fix spandex-secondary ENV

### PR DESCRIPTION
**Pull request by tkbrigham**
- [ ] styleCheck passes
- [ ] unit tests pass
- [ ] integration tests pass
- [ ] manual tests pass
- [ ] API docs updated

### CODE REVIEW

**Comment by tkbrigham** [comment]
@rlvoyer I've built and tested locally, and it makes it through the original errors fine but still has this error:
```
ScriptEngineManager providers.next(): javax.script.ScriptEngineFactory: Provider scala.tools.nsc.interpreter.IMain$Factory not found
```

I ran `sbt clean`, `./bin/build.sh`, and rebuild the container and still got the same thing. Is this as problematic as it looks?

File: spandex-secondary/docker/Dockerfile [hunk 1]
```diff
@@ -20,6 +20,12 @@ ENV JAVA_MAX_METASPACE 64m
 
 ENV SECONDARY_CONFIG secondary.conf
 
+# Environment settings (from spandex-http)
+ENV JAVA_XMX 512m
```
Comment (rlvoyer) [inline spandex-secondary/docker/Dockerfile]: This should be higher. The default in apps-marathon is 2816m, which we override in various envs. I'm inclined to put this at something a little more reasonable like 2g.

File: spandex-secondary/docker/Dockerfile [hunk 2]
```diff
@@ -20,6 +20,12 @@ ENV JAVA_MAX_METASPACE 64m
...
+ENV JMX_PORT 8049
```
Comment (rlvoyer) [inline spandex-secondary/docker/Dockerfile]: We override this in apps-marathon as well. If we can, we should make it consistent with that (6309).

File: spandex-secondary/docker/Dockerfile (hunk 1 above)
Comment (tkbrigham) [reply spandex-secondary/docker/Dockerfile]: 2g for local dev? Seems high to me, but happy to oblige. As the comment says, I lifted all of this directly from spandex-http.

File: spandex-secondary/docker/Dockerfile (hunk 2 above)
Comment (tkbrigham) [reply spandex-secondary/docker/Dockerfile]: 👍 good catch.

**Comment by tkbrigham** [comment]
🤞

**Pull request merged**